level/level.c

```diff
@@ -239,13 +249,13 @@ level* make_level(long int map_seed) {
     push_inventory(lvl->player, stick);
 
     for (int i=0; i < lvl->mob_count-1; i++) {
-        lvl->mobs[i]->x = rand()%(level_width-2) + 1;
-        lvl->mobs[i]->y = rand()%(level_height-2) + 1;
+        lvl->mobs[i]->x = choose(level_width - 1);
+        lvl->mobs[i]->y = choose(level_height - 1);
         lvl->mobs[i]->active = true;
 
-        //TODO magic number
-        switch (rand()%4) {
-            case 0:
+        //TODO magic number (not really, but still not good)
```

Comment:
What do you want? Eventually this will probably turn into:
```
monster_idx = choose(num_monster_types) -1; // -1 because you made choose 1 indexed
monster_builder_functions[monster_idx](lvl);
```
But unless you want to set that up, this is about as clear as it gets.

Reply:
I have not looked into what I want to do here, so you are likely correct.

Unrelated, seems like this would be a pretty great place for an enum though.